index.dd

```diff
@@ -53,6 +53,28 @@ void main()
 )
 )
 $(EXTRA_EXAMPLE
+----
+void main()
```

Comment:
Needs a comment at the top explaining what the program does.
"Counts frequencies of adjacent pairs of DNA bases" (though I'm fairly sure I got the terminology wrong).